### PR TITLE
doc/user: patch free trial FAQs

### PR DESCRIPTION
**Pull request by morsapaes**
We seem to frequently point prospects to the free trial FAQ page, which is slightly outdated. Please double-check my math!

Also took the chance to patch a few related typos.

### CODE REVIEW

File: doc/user/content/free-trial-faqs.md [hunk 1]
```diff
@@ -7,55 +7,75 @@ menu:
...
 ```sql
 SELECT sum(s.credits_per_hour) AS credit_consumption_rate
   FROM mz_cluster_replicas r
   JOIN mz_internal.mz_cluster_replica_sizes s ON r.size = s.size;
 ```
 
-For example, if you start your free trial with the [getting started guide](/get-started/), you'll end up consuming credits at a rate of `3.5` per hour:
+For example, if you start your free trial by following the [getting started guide](/get-started/quickstart),
+or if you otherwise only use the pre-installed `quickstart` cluster
+(`100cc`), you will end up consuming `1` credit per hour:
```
Comment (benesch) [inline doc/user/content/free-trial-faqs.md:56]:
```suggestion
(`25cc`), you will end up consuming `1` credit per hour:
```

(quickstarts will be `25cc` starting very soon)

**Review by benesch — APPROVED** [review]

File: doc/user/content/free-trial-faqs.md (hunk 1 above)
Comment (morsapaes) [reply doc/user/content/free-trial-faqs.md:56]: I'll adjust things then!

Comment (benesch) [reply doc/user/content/free-trial-faqs.md:56]: Er, sorry, as in probably later today: https://github.com/MaterializeInc/cloud/pull/9202

Comment (morsapaes) [reply doc/user/content/free-trial-faqs.md:56]: :le-sigh:

Comment (benesch) [reply doc/user/content/free-trial-faqs.md:56]: I'll patch in my fnlist PR.

**Pull request merged**